src/components/epicenter/JudgeAssignmentModal.tsx

```diff
@@ -111,42 +94,50 @@ const JudgeAssignmentModal = ({ visible, handleCancel }: JudgeTypes) => {
   };
 
   return (
-    <Modal visible={visible} onCancel={handleCancel} onOk={handleSubmit} okText="Create Assignment">
-      <Form>
-        <Title level={3}>Manual Assignment</Title>
-        <Select
-          style={{ width: 240 }}
-          showSearch
-          placeholder="Select a Project"
-          optionFilterProp="children"
-          filterOption={(input: any, option: any) =>
-            option?.children.toLowerCase().indexOf(input.toLowerCase()) >= 0
-          }
-          onChange={handleChange}
-        >
-          {data.map((project: Project) => (
-            <Option value={project.id} key={project.id}>
-              {project.name}
-            </Option>
-          ))}
-        </Select>
-        <Select
-          showSearch
-          placeholder="Select a Judge"
-          style={{ width: 240, marginTop: 10 }}
-          onChange={handleUserChange}
-          disabled={users.length === 0}
-          optionFilterProp="children"
-          filterOption={(input: any, option: any) =>
-            option?.children.toLowerCase().indexOf(input.toLowerCase()) >= 0
-          }
-        >
-          {users.map(user => (
-            <Option value={user.id} key={user.id}>
-              {user.name}
-            </Option>
-          ))}
-        </Select>
+    <Modal
+      open={open}
+      onCancel={handleCancel}
+      okText="Create Assignment"
+      onOk={onSubmit}
+      destroyOnClose
+    >
+      <Form form={form} layout="vertical" preserve={false}>
+        <Title level={3}>Manual Judge Assignment</Title>
+        <Form.Item name="project" label="Project" rules={[FORM_RULES.requiredRule]}>
+          <Select
+            style={{ width: 250 }}
+            showSearch
+            placeholder="Select a Project"
+            optionFilterProp="children"
+            filterOption={(input: any, option: any) =>
+              option?.children.toLowerCase().indexOf(input.toLowerCase()) >= 0
+            }
+          >
+            {projectsData.map((project: Project) => (
+              <Option value={project.id} key={project.id}>
+                {project.name}
+              </Option>
+            ))}
+          </Select>
+        </Form.Item>
+        <Form.Item name="user" label="Judge" rules={[FORM_RULES.requiredRule]}>
+          <Select
```

Comment:
I think this might be better as a multi-select

Reply:
This is the manual assign where u assign a certain judge to a project. Rn, the backend is only configured to assign one judge at a time, so i think its fine like this